MessageStackView/Messages/BadgeMessage/BadgeMessageView.swift

```diff
@@ -138,6 +138,11 @@ open class BadgeMessageView: UIView {
         containerView.layer.cornerRadius = layer.cornerRadius
         shadow = BadgeMessageView.shadowComponents
         
+        if #available(iOS 13.0, *) {
+            layer.cornerCurve = .continuous
+            containerView.layer.cornerRadius = layer.cornerRadius
```

Comment:
Can we remove this line as it's done above? Or must it come `layer.cornerCurve = .continuous`

Reply:
Sorry, meant to also set the cornerCurve!